### DISCUSSION TITLE
Where to find verbs, nouns and programs for luminary99?

### DISCUSSION THREAD

**Discussion by jumpjack** [virtualagc/virtualagc#1163] [Q&A] [answered]
I'd need the short descriptions of all verbs, nouns and programs in use in Apollo 11 LGC, where can I find them?

There are several sources available around, but each one says a different thing...

**Comment by rburkey2005** [comment, marked as answer]
So as to prevent anybody who might have an answer wasting effort by providing references that had already been consulted, here are the references I'm aware of that had already been provided prior to creation of this discussion thread:

- [The AC Electronics Manual for Apollo 11, beginning on PDF page 92](http://www.ibiblio.org/apollo/Documents/AcElectronicsApollo11.pdf#page=92)
- [The Apollo 11 Flight Crew G&N Dictionary](http://www.ibiblio.org/apollo/Documents/ApolloXI_FlightCrewG&NDictionary_5-29-1969.pdf).
- [The Quick-Reference Cards](http://www.ibiblio.org/apollo/links2.html#ReferenceCards).
- [The AGC program listing of the ASSEMBLY AND OPERATION INFORMATION module](https://www.ibiblio.org/apollo/listings/Luminary099/ASSEMBLY_AND_OPERATION_INFORMATION.agc.html);  also, [the EXTENDED VERBS module](https://www.ibiblio.org/apollo/listings/Luminary099/EXTENDED_VERBS.agc.html)
- [The GSOP document, section 4](http://www.ibiblio.org/apollo/links2.html#GSOPs)

Unfortunately, for some of these references we don't have the Apollo 11 version.

**Comment by rburkey2005** [comment]
Here are some additional thoughts:

- Perhaps you could be more specific about what discrepancies are bothering you.  As your question is phrased now, without any specifics, the best you could hope for is to get a link to yet some other document that will have discrepancies vs the ones that have already been proposed.
- In general, the most-authoritative source would be the GSOP.  You can think of it as the specification.
- Lacking that, I'd think the Flight Crew G&N Dictionary would be the most authoritative, because if you're giving the crew wrong information you've potentially got a very serious problem on your hands.
- The AC Manual is often the most helpful document, but it's the least authoritative.

**Comment by jumpjack** [comment]
Problem is the so many issues, revisions and versions of the same document, so it's nearly impossible to figure out which one exactly was the applicabile one on July, 16th 1969.
Maybe the _Flight Crew G&N Dictionary_  you mention is the best choice, given that it's specifically marked "LM-5" and was issued just a couple of months before launch.

**Comment by rburkey2005** [comment]
Yes, I'd think that the one specifically marked that way is likely to be the best reference.  More to the point, how could anyone today tell you otherwise?Sent on my Samsung Galaxy A51.
-------- Original message --------From: jumpjack ***@***.***> Date: 2/11/22  2:25 AM  (GMT-06:00) To: virtualagc/virtualagc ***@***.***> Cc: Ronald Burkey ***@***.***>, Comment ***@***.***> Subject: Re: [virtualagc/virtualagc] Where to find verbs, nouns and programs
  for luminary99? (Discussion #1163) 
Problem is the so many issues, revisions and versions of the same document, so it's nearly impossible to figure out which one exactly was the applicabile one on July, 16th 1969.
Maybe the Flight Crew G&N Dictionary  you mention is the best choice, given that it's specifically marked "LM-5" and was issued just a couple of months before launch.
